### DISCUSSION TITLE
New drawing app for kids based on fabric.js now available on brilliantstarmagazine.org

### DISCUSSION THREAD

**Discussion by sjscotti** [fabricjs/fabric.js#7740] [Show and tell]
Hi and thanks for developing such a great drawing library!

I was looking to add this info to the Wiki under "Who's using Fabric", but I didn't see how to do that, so I am using Discussions instead.

We had an old drawing program for kids on the https://brilliantstarmagazine.org/ website, but it was no longer supported by the original developer, and some of its capabilities had been "broken" by changes in the website over the years.  So when we learned about fabric.js, we decided to create an entirely new drawing program ourselves, primarily based on fabric.js, that expanded the capabilities that were in the old drawing program.  It hasn't been advertised much yet since a formal "launching" (that may include a contest where kids can use it and submit their creations) hasn't been announced.  But the application is live and available at 

https://brilliantstarmagazine.org/games/imagination-station2

Please take a look and provide any feedback (especially if you find bugs! ;-) )

**Comment by asturur** [comment]
Congratulation for your release!
